cvat-core/src/server-proxy.js

```diff
@@ -1120,6 +1120,77 @@
                 }
             }
 
+            async function createCloudStorage(storageDetail) {
+                const { backendAPI } = config;
+
+                try {
+                    const response = await Axios.post(`${backendAPI}/cloudstorages`, JSON.stringify(storageDetail), {
+                        proxy: config.proxy,
+                    });
+                    return response.data;
+                } catch (errorData) {
+                    throw generateError(errorData);
+                }
+            }
+
+            async function saveCloudStorage(id, cloudStorageData) {
+                const { backendAPI } = config;
+
+                try {
+                    await Axios.patch(`${backendAPI}/cloudstorages/${id}`, JSON.stringify(cloudStorageData), {
+                        proxy: config.proxy,
+                        headers: {
+                            'Content-Type': 'application/json',
+                        },
+                    });
+                } catch (errorData) {
+                    throw generateError(errorData);
+                }
+            }
+
+            async function getCloudStorages(filter = '') {
+                const { backendAPI } = config;
+
+                let response = null;
+                try {
+                    response = await Axios.get(`${backendAPI}/cloudstorages?page_size=20&${filter}`, {
+                        proxy: config.proxy,
+                    });
+                } catch (errorData) {
+                    throw generateError(errorData);
+                }
+
+                return response.data;
+            }
+
+            async function getCloudStorageContent(id, manifestPath) {
+                const { backendAPI } = config;
+
+                let response = null;
+                try {
+                    const url = manifestPath
+                        ? `${backendAPI}/cloudstorages/${id}/content?manifest=${manifestPath}`
+                        : `${backendAPI}/cloudstorages/${id}/content`;
```

Comment:
```suggestion
                    const url = `${backendAPI}/cloudstorages/${id}/content${manifestPath ? `?manifest=${manifestPath}` : ''}`;
```

Reply:
Based on how I implemented the [server side](https://github.com/openvinotoolkit/cvat/blob/142dea72033e56bf946c2384e74c37bd5fb8fa75/cvat/apps/engine/views.py#L1160), I would like to leave the option not to specify `manifest_path`

Reply:
You still have this option, I just showed you how you can do it using one line.

Reply:
Omm, sorry, I saw wrong at first...